src/components/instructor/EnrollmentsView.js

```diff
@@ -1,24 +1,93 @@
+
 import React, {useState, useEffect} from 'react';
 import {useLocation} from 'react-router-dom';
+import {SERVER_URL} from '../../Constants';
 
-// instructor view list of students enrolled in a section 
-// use location to get section no passed from InstructorSectionsView
-// fetch the enrollments using URL /sections/{secNo}/enrollments
-// display table with columns
-//   'enrollment id', 'student id', 'name', 'email', 'grade'
-//  grade column is an input field
-//  hint:  <input type="text" name="grade" value={e.grade} onChange={onGradeChange} />
+//// instructor view list of students enrolled in a section
+//// use location to get section no passed from InstructorSectionsView
+//// fetch the enrollments using URL /sections/{secNo}/enrollments
+//// display table with columns
+////   'enrollment id', 'student id', 'name', 'email', 'grade'
+////  grade column is an input field
+////  hint:  <input type="text" name="grade" value={e.grade} onChange={onGradeChange} />
 
 const EnrollmentsView = (props) => {
+    const [enrollments, setEnrollments] = useState([]);
+    const [message, setMessage] = useState('');
+    const headers = ['EnrollmentId', 'StudentId', 'Name', 'Email', 'Grade'];
+
 
     const location = useLocation();
+
     const {secNo, courseId, secId} = location.state;
 
-    return(
-        <> 
-            <h3>Not implemented</h3>   
-        </>
-    );
-}
+
+  //  const {secNo} = location.state;
+
+    useEffect(() => {
+        fetchEnrollments(secNo);
+    }, [secNo]);
```

Comment:
I dont think you need to pass `[secNo]` here. This is a global var in this component

Reply:
I can check what's going on. @JADuarte18 were you able to test this file? Don't want to make any changes yet if you working on it.